models/balancer/balancer_trades.sql

```diff
@@ -0,0 +1,51 @@
+{{ config(
+    schema = 'balancer',
+    alias = 'trades',
+    post_hook='{{ expose_spells(\'["arbitrum", "ethereum", "optimism", "polygon"]\',
+                                "project",
+                                "balancer",
+                                \'["bizzyvinci"]\') }}'
+    )
+}}
+
+{% set balancer_models = [
+    ref('balancer_arbitrum_trades'),
+    ref('balancer_ethereum_trades'),
+    ref('balancer_optimism_trades'),
+    ref('balancer_polygon_trades')
+] %}
+
+
+SELECT *
+FROM (
+    {% for dex_model in balancer_models %}
+    SELECT
+        blockchain,
+        project,
+        version,
+        block_date,
+        block_time,
+        token_bought_symbol,
+        token_sold_symbol,
+        token_pair,
+        token_bought_amount,
+        token_sold_amount,
+        CAST(token_bought_amount_raw AS DECIMAL(38,0)) AS token_bought_amount_raw,
```

Comment:
please remove these `cast()` statements in the rollup view here, rather ensuring they are used in the base model this reads from -- i.e. `balancer_ethereum_trades` and other blockchains

Reply:
I will note that cast() only apply/use in the base models in the future.